plugins/bundle/plugin.go

```diff
@@ -18,10 +18,11 @@ import (
 	"sync"
 	"time"
 
+	bundleUtils "github.com/open-policy-agent/opa/internal/bundle"
+
```

Comment:
[nit] previous location of this line was OK, I think

Reply:
Updated